warehouse/README.md

```diff
@@ -308,3 +308,19 @@ In addition to the steps specified in the dbt docs, [Google Private Access was e
 and the cal-itp-data-infra-staging project's default service account (`473674835135-compute@developer.gserviceaccount.com`) was granted access to the production project
 since the buckets for compiled Python models (`gs://calitp-dbt-python-models` and `gs://test-calitp-dbt-python-models`)
 as well as external tables exist in the production project.
+
+## Testing Warehouse Image Changes
+
+A person with Docker set up locally can build a development version of the underlying warehouse image at any time after making changes to the Dockerfile or its requirements. From the relevant subfolder, run
+
+```bash
+docker build -t ghcr.io/cal-itp/data-infra/warehouse:development .
+```
+
+That image can be used alongside [a local Airflow instance](../airflow/README.md) to test changes locally prior to merging.
```

Comment:
Is it possible to link to the specific subsection about testing PodOperators?

Reply:
Addressed in [ac52c02](https://github.com/cal-itp/data-infra/pull/3156/commits/ac52c0287d433386871beca0c45af14ede6b1f84)